test/integ_tests/requirements.txt

```diff
@@ -11,7 +11,7 @@ PennyLane_Lightning<=0.32
 pytest
 pytest-rerunfailures
 pytest-xdist
-qiskit_braket_provider
+qiskit_braket_provider==0.0.4
```

Comment:
Can you add why do we have to pin this?

Reply:
I am not sure what the issue is. Builds are failing so I was doing this to test that the previous versions are alright.